js/src/secureApi.js

```diff
@@ -26,6 +26,7 @@ export default class SecureApi extends Api {
     this._connectState = sysuiToken === 'initial' ? 1 : 0;
     this._needsToken = false;
     this._dappsPort = 8080;
+    this._dappsInterface = null;
```

Comment:
Agreed. Had it that way around as a first stab. Just wanted to have additional protection against the case if we ever get null back for whatever reason. (Which would point to more serious issues, but ...)